utils/S3UtilsMongoClient.js

```diff
@@ -74,6 +74,92 @@ class S3UtilsMongoClient extends MongoClientInterface {
         }
     }
 
+    async updateInflightDeltas(allMetrics, log) {
+        let cursor;
+        try {
+            const bucketKeys = [];
+
+            if (!allMetrics || !Array.isArray(allMetrics) || allMetrics.length === 0) {
+                return allMetrics;
+            }
+
+            const usersBucketCreationDatesMap = await this._getUsersBucketCreationDates(log);
+            Object.keys(usersBucketCreationDatesMap).forEach(bucketEntry => {
+                const bucketName = bucketEntry.split(constants.splitter)[1];
+                const bucketKey = `bucket_${bucketName}_${new Date(usersBucketCreationDatesMap[bucketEntry]).getTime()}`;
+                bucketKeys.push(bucketKey);
+            });
```

Comment:
nit:
```suggestion
            const bucketKeys = Object.keys(usersBucketCreationDatesMap).map(bucketEntry => {
                const bucketName = bucketEntry.split(constants.splitter)[1];
                return `bucket_${bucketName}_${new Date(usersBucketCreationDatesMap[bucketEntry]).getTime()}`;
            });
```

Reply:
resolving as the logic is removed due to the change you suggested below